### ISSUE TITLE
Pretraining and Finetuning Documentation

### ISSUE THREAD

**Issue by strasserpatrick** [MIC-DKFZ/nnUNet#2368]
Hello, 

I am not sure if this part of the pretrain-and-finetune docs is consistent and correct. Can you please clarify this for me and say if my thinking is wrong:

(from the docs:)
Now we can take the plans from the target dataset and transfer it to the pretraining dataset:

```bash
nnUNetv2_move_plans_between_datasets -s PRETRAINING_DATASET -t TARGET_DATASET -sp PRETRAINING_PLANS_IDENTIFIER -tp TARGET_PLANS_IDENTIFIER
```

`PRETRAINING_PLANS_IDENTIFIER` is hereby probably nnUNetPlans unless you changed the experiment planner in 
nnUNetv2_plan_and_preprocess. For `TARGET_PLANS_IDENTIFIER` we recommend you set something custom in order to not 
overwrite default plans.

I understand the code of `nnUNetv2_move_plans_between_datasets` as copying the plans file from source (`-s`and `-sp`) to target (`-t`and `-tp`). We want to copy the plans file from finetuning to pretraining dataset. For me, the move-plans command needs to be as followed:

```bash
nnUNetv2_move_plans_between_datasets -s TARGET_DATASET -t PRETRAINING_DATASET -sp TARGET_PLANS_IDENTIFIER -tp PRETRAINING_PLANS_IDENTIFIER
```

With the following section also being reversed: 
`TARGET_PLANS_IDENTIFIER` is hereby probably nnUNetPlans unless you changed the experiment planner in 
nnUNetv2_plan_and_preprocess. For `PRETRAINING_PLANS_IDENTIFIER` we recommend you set something custom in order to not overwrite default plans.

Am I right? We want to move the plan of the finetuning task to be moved to the pretraining dataset => source for moving the plan is finetuning, target for plan moving is pretraining dataset.

I can do the docs changes if my thinking is correct. Please explain to me if not :)

Thank you!

**Comment by strasserpatrick** [comment]
my changes would then also be consistent with the next step of preprocessing the pretraining dataset:

```bash
nnUNetv2_preprocess -d PRETRAINING_DATASET -plans_name TARGET_PLANS_IDENTIFIER
```

**Comment by TaWald** [comment]
In our experience you should preprocess the target (downstream) dataset the same way as was used in your pretraining.
Hence you should use the `PRETRAINING_PLANS_IDENTIFIER` in your downstream task as well, which would make the docu correct. 

This is due to the model having learned all the "knowledge" in the pertaining with a certain intensity range (normalization) and geometry (spacing + patch size). If you get a shift you may lose pre-training gains which is undesirable.
